### ISSUE TITLE
Training on older versions of pytorch

### ISSUE THREAD

**Issue by bmazoure** [wohlert/semi-supervised-pytorch#4]
When training with older versions of pytorch (the one that does not still have the nn.funcitonal.softmax(dim) for example), the training stagnates around 0.10-0.12 accuracy for the first 50 epochs. 

Do you have any idea why this might happen?

**Comment by wohlert** [comment]
I am currently rewriting the implementation (on the master branch) so everything is a mess until the release of version 1.

**Comment by bmazoure** [comment]
Okay!
